packages/babel-helper-explode-assignable-expression/src/index.ts

```diff
@@ -49,20 +66,20 @@ function getPropRef(node, nodes, file, scope) {
 }
 
 export default function (
-  node: Object,
-  nodes: Array<Object>,
-  file,
+  node: Identifier | MemberExpression,
+  nodes: Array<AssignmentExpression>,
+  file: void,
   scope: Scope,
   allowedSingleIdent?: boolean,
 ): {
-  uid: Object,
-  ref: Object,
+  uid: Identifier | MemberExpression;
+  ref: Identifier | MemberExpression;
 } {
   let obj;
   if (t.isIdentifier(node) && allowedSingleIdent) {
     obj = node;
   } else {
-    obj = getObjRef(node, nodes, file, scope);
```

Comment:
`file` is unused. I leave it in public interface for compatibility.

Reply:
Maybe add a `TODO(Babel 8)`, so we don't forget about removing it.